apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -528,70 +523,83 @@ static void utc_audio_pcm_set_config_tc_n(void)
 	size = pcm_get_buffer_size(g_pcm);
-	g_byte_size = pcm_frames_to_bytes(g_pcm, size);
-	TC_ASSERT_GT("pcm_frames_to_bytes", g_byte_size, 0);
+	TC_ASSERT_GT("pcm_get_buffer_size", size, 0);
+
+	frame_size = pcm_bytes_to_frames(g_pcm, size);
+	TC_ASSERT_GT("pcm_bytes_to_frame", frame_size, 0);
 	TC_SUCCESS_RESULT();
 }
 
 /**
-* @testcase         audio_pcm_frame_to_bytes_n
-* @brief            convert frames to bytes
-* @scenario         get configuration value of NULL pcm and calculate frame to byte
-* @apicovered       pcm_frames_to_bytes
+* @testcase         audio_pcm_bytes_to_frames_n
+* @brief            convert bytes to frame
+* @scenario         get configuration value of NULL pcm and calculate byte to frame
+* @apicovered       pcm_bytes_to_frames
 * @precondition     NA
 * @postcondition    NA
 */
-static void utc_audio_pcm_frames_to_bytes_n(void)
+static void utc_audio_pcm_bytes_to_frames_n(void)
 {
-	unsigned int bytes;
-	bytes = pcm_frames_to_bytes(NULL, pcm_get_buffer_size(g_pcm));
-	TC_ASSERT_EQ("pcm_frames_to_bytes", bytes, 0);
+	unsigned int frame;
+	frame = pcm_frames_to_bytes(NULL, 0);
+	TC_ASSERT_LEQ("pcm_frames_to_bytes", frame, 0);
 	TC_SUCCESS_RESULT();
 }
 
 /**
-* @testcase         audio_pcm_bytes_to_frames_p
-* @brief            convert bytes to frame
-* @scenario         get configuration value of pcm and calculate byte to frame
-* @apicovered       pcm_bytes_to_frames
+* @testcase         audio_pcm_frame_to_bytes_p
+* @brief            convert frames to bytes
+* @scenario         get configuration value of pcm and calculate frame to byte
+* @apicovered       pcm_frames_to_bytes
 * @precondition     pcm should be opened before.
 * @postcondition    NA
 */
-static void utc_audio_pcm_bytes_to_frames_p(void)
+static void utc_audio_pcm_frames_to_bytes_p(void)
 {
-
+	ssize_t size;
+	unsigned int bytes;
 	unsigned int frame_size;
-	frame_size = pcm_bytes_to_frames(g_pcm, g_byte_size);
-	TC_ASSERT_GT("pcm_bytes_to_frame", frame_size, 0);
+
+	/* set basic configuration values for next test */
+	pcm_set_config(g_pcm, NULL);
+	size = pcm_get_buffer_size(g_pcm);
+	TC_ASSERT_GT("pcm_get_buffer_size", size, 0);
+
+	frame_size = pcm_bytes_to_frames(g_pcm, size);
+	TC_ASSERT_GT("pcm_bytes_to_frames", frame_size, 0);
+
+	bytes = pcm_frames_to_bytes(g_pcm, frame_size);
+	TC_ASSERT_GT("pcm_frames_to_bytes", bytes, 0);
 	TC_SUCCESS_RESULT();
 }
 
 /**
-* @testcase         audio_pcm_bytes_to_frames_n
-* @brief            convert bytes to frame
-* @scenario         get configuration value of NULL pcm and calculate byte to frame
-* @apicovered       pcm_bytes_to_frames
+* @testcase         audio_pcm_frame_to_bytes_n
+* @brief            convert frames to bytes
+* @scenario         get configuration value of NULL pcm and calculate frame to byte
+* @apicovered       pcm_frames_to_bytes
 * @precondition     NA
 * @postcondition    NA
 */
-static void utc_audio_pcm_bytes_to_frames_n(void)
+static void utc_audio_pcm_frames_to_bytes_n(void)
```

Comment:
Here's only place that calling api wrongly.
Please check title & api call in modified code

Reply:
Fixed